website/blog/2023-06-14-install-ferretdb-mongodb-alternative-ubuntu.mdx

```diff
@@ -0,0 +1,321 @@
+---
+slug: install-ferretdb-mongodb-alternative-ubuntu
+title: 'How to Install FerretDB on Ubuntu'
+author: Klinsmann Öteyo
+description: >
+  This is a guide on how to install FerretDB - the MongoDB alternative - on Ubuntu.
+image: /img/blog/install-ferretdb-ubuntu.jpg
+keywords: [FerretDB, mongodb alternative]
```

Comment:
Do we have guidelines for tags and keywords? Why, for example, we don't use tags there? What keywords "FerretDB" and "mongodb alternative" are there, given that pretty much all blog posts are about FerretDB which is MongoDB (not "mongodb") alternative?

Reply:
Didn't add tags for this because I didn't know which tags best fit it since it's an installation guide (maybe "installation guide" 🤔). 

For the keyword, they are mostly case-insensitive; this is a repost though, so won't matter all that much. I've added some better user-specific keywords for it.

I do agree there should be a structure to the tags and keywords and created an issue to address that: 
https://github.com/FerretDB/FerretDB/issues/2847